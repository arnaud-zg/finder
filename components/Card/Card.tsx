import React from 'react'
import { style } from 'typestyle'
import { rem } from 'csx'
import { BoxFunction, BoxUnit } from 'csstips'
import { colors } from '../../constants/colors'

const className = style({
  border: 'solid',
  borderColor: colors.gray.toHexString(),
  borderWidth: rem(0.0625),
  marginTop: rem(1),
})

export class Card extends React.PureComponent {
  render() {
    const { children } = this.props
    return <div className={className}>{children}</div>
  }
}
