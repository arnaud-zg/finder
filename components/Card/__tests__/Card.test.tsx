import React from 'react'
import renderer from 'react-test-renderer'
import { Card } from '../Card'

describe('Card', () => {
  it('renders correctly', () => {
    const testRenderer = renderer.create(<Card>Hello World</Card>)
    expect(testRenderer.toJSON()).toMatchSnapshot()
  })
})
