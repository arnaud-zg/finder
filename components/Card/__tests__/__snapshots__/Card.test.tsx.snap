// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<div
  className="f1siatag"
>
  Hello World
</div>
`;

exports[`Card should take a snapshot of CardIndex 1`] = `
Object {
  "Card": [Function],
}
`;
